### PR TITLE
Added the azure volume unattached

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Added the azure volumes_unattached policy.
1. Retrieve the azure volumes
2. Filter the unattached volumes
3. verify and delete the resource
4. Upload to the Elasticsearch

Screenshot 2024-01-03 at 1 48 32 PM
Screenshot 2024-01-03 at 1 48 32 PM

### CODE REVIEW

**Review by ebattat — APPROVED** [review]
/LGTM

File: cloud_governance/main/main_oerations/main_operations.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def run(self):
         policy_runner = self.get_policy_runner()
         for policy_type, policies in policies_list.items():
             # @Todo support for all the aws policies, currently supports ec2_run as urgent requirement
-            if self._policy in policies and self._policy in "instance_run":
+            if self._policy in policies and self._policy in ["instance_run", "unattached_volume"]:
```
Comment (ebattat) [inline cloud_governance/main/main_oerations/main_operations.py:41]: Is it going to change inthe future, if yes add todo comment?

Comment (athiruma) [reply cloud_governance/main/main_oerations/main_operations.py:41]: ok

Comment (athiruma) [reply cloud_governance/main/main_oerations/main_operations.py:41]: Already have the comment above.

**Pull request merged**